Managers/SaveManager.gd

```diff
@@ -4,6 +4,8 @@ var save_file: ConfigFile
 
 func _ready() -> void:
 	save_file = ConfigFile.new()
+	if !DirAccess.dir_exists_absolute("user://save/"):
+		DirAccess.make_dir_absolute("user://save/")
```

Comment:
this is not recursive so if we ever want a more nested structure we'll need to think of using `make_dir_recursive` instead. I mean, we could directly use it here, but idk if it's really useful